### PR TITLE
adjust minimum cold on space helmets and add (some) doc

### LINKED ISSUE #64398: EVA helmets and related helmets give full temperature immunity even with nothing else on
## Round ID:
tested on private

## Reproduction:
put on EVA helmet, you now have full temperature resist
reak
I'm pretty sure this is an issue with the define SPACE_HELM_MIN_TEMP_PROTECT, as other helmets seem to work as intended. However, other helmets using that define gain full temp immunity even if they only cover the head.

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

before, you were able to wear a space helmet out into space completely naked, and not be cold at all. this adjusts the SPACE_HELM_MIN_TEMP_PROTECT from 2.0 (space temperatures) to 72 (same as suit). when the thermal regulator on a suit is enabled, you return to have full cold protection.

fixes #64398

## Why It's Good For The Game

refer to branch name

## Changelog

:cl:
fix: fixed space helmet temperature bounds
/:cl:

### CODE REVIEW

File: code/__DEFINES/atmospherics/atmos_mob_interaction.dm [hunk 1]
```diff
@@ -90,8 +90,12 @@
 
 //CLOTHES
 
+// Welcome to Tsu's explanation on: what is this shit? Basically, the minimum temp protection values dictates the absolute lowest of a temperature someone can handle before they
+// start taking temperature damage. The way it is setup makes it kind of seem like its "bounds" for what the temperature protection possibly "could" be, but nope!
+// Its what it is. Weird. Don't set space helmet temperature protections to 2.0, because then you can go naked into space and not be cold. This is bad.
+
 /// what min_cold_protection_temperature is set to for space-helmet quality headwear. MUST NOT BE 0.
-#define SPACE_HELM_MIN_TEMP_PROTECT 2.0
+#define SPACE_HELM_MIN_TEMP_PROTECT 72
```
Comment (Fikou) [inline code/__DEFINES/atmospherics/atmos_mob_interaction.dm:98]: make this a separate define. cult hoods, marine helmets and plasmaman helmets rely on this.

Comment (Fikou) [reply code/__DEFINES/atmospherics/atmos_mob_interaction.dm:98]: nevermind! its good

**Review by Fikou — APPROVED** [review]
lgtm :)

**Comment by ExcessiveUseOfCobblestone** [comment]
Please doc it properly let me get the saved reply

You have generated new variables/procs/defines without proper documentation for the autodoc to work.

Please click [here ](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)for a guide on how to properly document these items.

**Comment by ExcessiveUseOfCobblestone** [comment]
Also why not just make it whatever the suit off define is if that’s what you’re doing with the hard value?

**Comment by magatsuchi** [comment]
> Also why not just make it whatever the suit off define is if that’s what you’re doing with the hard value?

so i don’t have to go back and edit all the code that already includes this for the helmet value

**Comment by ExcessiveUseOfCobblestone** [comment]
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

Also I’m not asking you to change the define itself, I’m saying set the helmet define’s value to the define of the suit (you will have to reorder pretty sure)

**Comment by ExcessiveUseOfCobblestone** [comment]
in my divine knowledge i forgot the obvious being you could just keep the number here and define /w define below

so

```DM
#define SPACE_HELM_MIN_TEMP_PROTECT 72
...
...
...
#define SPACE_SUIT_MIN_TEMP_PROTECT_OFF SPACE_HELM_MIN_TEMP_PROTECT
```

will also work

**Comment by LemonInTheDark** [comment]
This isn't an issue with space suits, it's an issue with body temperature. 
Space helmets will at most protect you from 30% of the cold of space, but that's enough of an offset to space's cold to make your natural bodytemp powerful enough to keep you above the board.

I get the feeling body temperature could do with a re-rework, moving it to something heat based rather then just temperature and scaling ratios. Sorry I didn't mention this earlier, only just noticed

**Pull request closed without merging**